example_proj/settings.py

```diff
@@ -122,6 +122,7 @@
     # 'django.contrib.admindocs',
 
     'django_behave',
+    'example_proj.apps.subdirectory_app',
```

Comment:
For readability, could this be a few lines up so that it is together with the example_app?